charts/akeyless-api-gateway/values.yaml

```diff
@@ -396,9 +399,11 @@ cachingConf:
 
   clusterCache:
     enabled: false
+    cacheAddress: ""
```

Comment:
remove cacheAddress - the client should not be exposed to it.. its internal service

Reply:
the customer wants to connect to his aws elasticache instance

Reply:
product did not approve this... who told you to do it? this is not a valid request

Reply:
eldad?